src/pam_unix_ng-common.c

```diff
@@ -25,6 +25,16 @@ parse_args(pam_handle_t *pamh, int flags, int argc, const char **argv,
   cfg->fail_delay = 2000;
   cfg->minlen = 8;
 
+  /*
+   * strictly spoken this reading of the logindefs has nothing to do with
+   * parsing PAM module arguments. it is rather a kind of default
+   * initialization from system file settings.
```

Comment:
That's not fully correct. It's the default if no option is specified and if you split this, you lost later the overview who is setting what.
So we have everything in one place.

Reply:
And I'm pretty sure that we will see very quickly feature requests for which we need to move that then back again here.